scenario/state.py

```diff
@@ -140,6 +140,69 @@ def copy(self) -> "Self":
         return copy.deepcopy(self)
 
 
+class CloudAuthType(str, Enum):
+    access_key_auth_type = "access-key"
+    instance_role_auth_type = "instance-role"
+    user_pass_auth_type = "userpass"
+    o_auth1_auth_type = "oauth1"
+    o_auth2_auth_type = "oauth2"
+    json_file_auth_type = "jsonfile"
+    client_certificate_auth_type = "clientcertificate"
+    http_sig_auth_type = "httpsig"
+    interactive_auth_type = "interactive"
+    empty_auth_type = "empty"
```

Comment:
Enum members are typically UPPERCASE in Python.

Reply:
Two other things:

1) We don't need the `_auth_type` suffix, as the enum class acts as a namespace: `CloudAuthType.USER_PASS` instead of `CloudAuthType.user_pass_auth_type`.
2) What's the benefit of inheriting from `str` and `Enum`? Not saying we shouldn't just wondering what it gains us.